### PR TITLE
LPS-137083 align the SLA status icon

### PR DESCRIPTION
**Pull request by jotascript**
Issue: [LPS-137083](https://issues.liferay.com/browse/LPS-137083)

### CODE REVIEW

**Comment by ramonmsp** [comment]
Just started reviewing :)

File: modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/css/_table.scss [hunk 1]
```diff
@@ -50,7 +51,7 @@
 	th:last-child,
 	td + td,
 	th + th {
-		padding-right: 1rem;
+		// padding-right: 1rem;
```
Comment (ramonmsp) [inline modules/dxp/apps/portal-workflow/portal-workflow-metrics-web/src/main/resources/META-INF/resources/css/_table.scss:54]: hey, @jotascript could you remove this comment?

**Review by ramonmsp — CHANGES_REQUESTED** [review]

**Review by naraandrade — APPROVED** [review]

**Comment by jotascript** [comment]
ci:test:relevant

**Comment by jotascript** [comment]
ci:forward

**Pull request closed without merging**